File: src/main/kotlin/top/charjin/shoppingserver/mapper/OsCollectionMapper.java
package top.charjin.shoppingserver.mapper;

import top.charjin.shoppingserver.entity.OsCollection;

public interface OsCollectionMapper {
    int deleteByPrimaryKey(Integer id);

    int insert(OsCollection record);

    int insertSelective(OsCollection record);

    OsCollection selectByPrimaryKey(Integer id);

    int updateByPrimaryKeySelective(OsCollection record);

    int updateByPrimaryKey(OsCollection record);
}

File: src/main/kotlin/top/charjin/shoppingserver/mapper/OsCollectionMapper.kt
package top.charjin.shoppingserver.mapper

import top.charjin.shoppingserver.entity.OsCollection

interface OsCollectionMapper {
    fun deleteByPrimaryKey(id: Int?): Int

    fun insert(record: OsCollection): Int

    fun insertSelective(record: OsCollection): Int

    fun selectByPrimaryKey(id: Int?): OsCollection

    fun updateByPrimaryKeySelective(record: OsCollection): Int

    fun updateByPrimaryKey(record: OsCollection): Int
}